modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserAccountResourceImpl.java

```diff
@@ -1606,6 +1612,19 @@ else if (user != null) {
 		return true;
 	}
 
+	private boolean _isMale(UserAccount.Gender gender, boolean male) {
```

Comment:
can we call the boolean male -> boolean defaultValue ? (and in case you need to switch the two parameters to have them sorted)